pyeudiw/oauth2/dpop.py

```diff
@@ -5,6 +5,7 @@
 from pydantic import BaseModel, HttpUrl
 from pyeudiw.jwt import JWSHelper
 from pyeudiw.jwt.utils import unpad_jwt_payload, unpad_jwt_header
+from pyeudiw.oauth2.exceptions import KidError
```

Comment:
please create/move KidError from oauth2 to the submodule jwk, and import then accordingly here